### DISCUSSION TITLE
The CLI for linux-arm64 doesn't work in termux on Android

### DISCUSSION THREAD

**Discussion by Yassinproweb** [tailwindlabs/tailwindcss#15338] [Help]
**What version of Tailwind CSS are you using?**

3.4.16

**What build tool (or framework if it abstracts the build tool) are you using?**

Vite

**What version of Node.js are you using?**

For example: v22.8.0

**What browser are you using?**

Brave browser

**What operating system are you using?**

Android

**Comment by philipp-spiess** [comment]
@Yassinproweb Hey! Can you elaborate what "does't work" mean? Termux seems to be a terminal emulator so if a program that works on Linux doesn't work on the emulator, it might be an issue with the terminal emulator. In either case, a repro and some more information would help.

**Comment by lost22git** [comment]
same issue.

```
❯ la ./build/.lustre/bin/tailwind
0755 40M u0_a459 2025-01-26 21:22   ./build/.lustre/bin/tailwind

hello/gleam/lustre_demo on  main [✘!?] via ⭐ v1.7.0 
❯ ./build/.lustre/bin/tailwind
Bad system call
```

**Comment by IzumiSenaSora** [comment]
Is there any news about it? If need any logs or anything i can help but at least fix this issue if possible

**Reply by IzumiSenaSora** [reply]
Tailwindcss cli "v3.4.17" file name "tailwindcss-linux-arm64" works just fine but anything from "v4" don't want to execute
